config/base/provider.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
```

Comment:
We need the `argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true` annotation set on this resource so ArgoCD can successfully sync the required CRD first.

Reply:
Added the annotation.